src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/custom.py

```diff
@@ -458,3 +472,48 @@ def return_valid_duration_create(update_value):
                 return None
     else:
         return None
+
+
+# NetwrokRuleSet Region
+def cli_networkrule_createupdate(client, resource_group_name, namespace_name, subnet=None, ip_mask=None, ignore_missing_vnet_service_endpoint=False, action='Allow'):
+    from azure.mgmt.servicebus.models import NWRuleSetVirtualNetworkRules, Subnet, NWRuleSetIpRules
+    netwrokruleset = client.get_network_rule_set(resource_group_name, namespace_name)
+
+    if netwrokruleset.virtual_network_rules is None:
+        netwrokruleset.virtual_network_rules = [NWRuleSetVirtualNetworkRules]
```

Comment:
Not sure what's going on here, but this doesn't look like it would be doing what you want at all. I think this should just be empty list `[]`

Reply:
corrected.